include/usb/usb_common.h

```diff
@@ -63,6 +63,7 @@
 #define USB_HID_DESC                    0x21
 #define USB_HID_REPORT_DESC             0x22
 #define USB_DFU_FUNCTIONAL_DESC         0x21
+#define USB_ASSOCIATION_DESC		0x0B
 
```

Comment:
is this line aligned?

Reply:
Actually I would not mind having patch converting spaces to tabs in those headers.

Reply:
aligned and using tabs, checkpatch is also happy

Reply:
Yes, you are using tabs, but the file "include/usb/usb_common.h" is using spaces, so it does not look consistent, everything should be in the same style, tabs or spaces (preferably tabs).

Reply:
But that one nitpick, maybe I will convert all those spaces to tabs afterwards, I made myself this mistake mixing tabs with spaces for USB version patch.

Reply:
I have zephyrized usb_common.h, see b1f1fe4, will squash if there are no objections.